### DISCUSSION TITLE
Fire emblem three houses test

### DISCUSSION THREAD

**Discussion by Alexsilvachief** [skyline-emu/skyline#1348] [Game Requests]
![Screenshot_20220519-001510](https://user-images.githubusercontent.com/87319404/169223748-083d87a6-1d9e-4c47-88f8-e0a8477d6e3f.png)
Just testing fire emblem three houses, unplayable only loads this and freezes

**Comment by Chibuzo2003** [comment]
U ppl shd also do smth Abt that I mean we are tired of playing lower graphics game at least do smth Abt the switch

**Reply by rebornsunny2000** [reply]
i have an idea:
why don't you play the *high graphic* games on your switch till this emulator progresses at the glorious speed it is already in.
not that graphics make a game anyway, this emulator already plays some masterpieces like Celeste and Sonic Mania at full speed, and is slowly improving 3D as well.
Stop whining and complaining please, appreciate the developers and the team, who are working this hard while they could literally be doing anything else, and in Tahlreth's words: "You are not paying for this emulator, it is free, so stop complaining."
Have a good day!

P.S Fire Emblem 3 Houses just showing that much is already a huge milestone!
